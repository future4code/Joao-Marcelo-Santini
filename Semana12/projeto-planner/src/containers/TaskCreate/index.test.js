import React from "react";
import renderer from "react-test-renderer";
import { TaskCreate } from "./index";

describe("Teste TaskCreate", () =>{
    test("Snapshot TaskCreate", () => {
        const snapTaskCreate = renderer.create(<TaskCreate />)
        .toJSON();
        expect(snapTaskCreate).toMatchSnapshot();
    });

});