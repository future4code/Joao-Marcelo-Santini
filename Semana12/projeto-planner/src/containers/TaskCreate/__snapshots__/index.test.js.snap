// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste TaskCreate Snapshot TaskCreate 1`] = `
<div
  className="sc-bdVaJa kmDWeR"
>
  <form
    className="sc-bwzfXH fTtZJQ"
    onSubmit={[Function]}
  >
    <div
      className="MuiFormControl-root-1"
    >
      <label
        className="MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13"
        data-shrink={false}
      >
        Tarefa
      </label>
      <div
        className="MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-formControl-37 MuiInput-formControl-24"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          className="MuiInputBase-input-46 MuiInput-input-31"
          disabled={false}
          name="text"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
          value=""
        />
      </div>
    </div>
    <br />
    <select
      className="sc-htpNat bqGasI"
      name="day"
      onChange={[Function]}
      value=""
    >
      <option
        selected="selected"
        value=""
      >
        Escolha o dia
      </option>
      <option
        value="Segunda"
      >
        Segunda-Feira
      </option>
      <option
        value="Terça"
      >
        Terça-Feira
      </option>
      <option
        value="Quarta"
      >
        Quarta-Feira
      </option>
      <option
        value="Quinta"
      >
        Quinta-Feira
      </option>
      <option
        value="Sexta"
      >
        Sexta-Feira
      </option>
      <option
        value="Sabado"
      >
        Sábado
      </option>
      <option
        value="Domingo"
      >
        Domingo
      </option>
    </select>
    <br />
    <button
      className="MuiButtonBase-root-79 MuiButton-root-53 MuiButton-text-55 MuiButton-flat-58"
      disabled={false}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="submit"
    >
      <span
        className="MuiButton-label-54"
      >
        Cadastrar
      </span>
      <span
        className="MuiTouchRipple-root-82"
      />
    </button>
  </form>
</div>
`;
